src/components/data-exploration/embedding/Embedding.jsx

```diff
@@ -207,7 +214,8 @@ const Embedding = (props) => {
   const updateCellsSelection = (selection) => {
     if (Array.from(selection).length > 0) {
       setCreateClusterPopover(true);
-      setSelectedIds(selection);
+      const selectedIdsToInt = new Set(Array.from(selection).map((id) => parseInt(id, 10)));
```

Comment:
Why are the cellIds not int in the first place?

Reply:
yep, vitessce returns strings